server/src/test/resources/logback-test.xml

```diff
@@ -7,14 +7,14 @@
     </encoder>
   </appender>
   <logger name="org.candlepin.util.filters.DumpFilter" level="INFO"/>
-  <logger name="org.candlepin" level="DEBUG"/>
+  <logger name="org.candlepin" level="INFO"/>
```

Comment:
We want debug output enabled by default during testing, right?

Reply:
It's awfully noisy.  I'm not opposed but maybe run it with DEBUG on your end and see what you think :)

Reply:
You're not wrong about it being noisy, but do we have a way of running the tests in debug mode without having to manually update this file? IIRC, it used to be INFO as it is here, but then it became DEBUG due to someone (probably me) consciously changing it or forgetting to change it back and it slipped through a review. Also, with the sheer amount of spam from just the raw test results alone, is the extra/reduced noise even significant?

I suppose with debugging being easier with with the provided IntelliJ licenses this is less important as it was years ago, I just would like this to be easier to flip on demand.

Reply:
This doesn't seem to make a difference with gradle. Disregard.